.gitignore

```diff
@@ -0,0 +1,15 @@
+/.bundle/
+/.yardoc
+/_yardoc/
+/coverage/
+/doc/
+/pkg/
+/spec/reports/
+/tmp/
+/log/
+.DS_Store
```

Comment:
:confounded: this file is in GitHub 
![image](https://user-images.githubusercontent.com/11413458/68333526-6fa01a00-00b7-11ea-9c99-bb633fc24f76.png)